packages/create-discord-bot/template/TypeScript/events/index.ts

```diff
@@ -0,0 +1,7 @@
+import type { ClientEvents } from 'discord.js';
+
+export interface Event<T extends keyof ClientEvents> {
+	execute(...parameters: ClientEvents[T]): Promise<void> | void;
+	name: T;
+	once?: boolean;
+}
```

Comment:
Could we either move this file to a `typings` directory or rename it? I think `index.ts` isn't a fitting name for just an  interface